src/pages/standup/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { FC, useState } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+
+import { standupUpdateType } from '@/interfaces/standup.type';
+import { FormatDate } from '@/utils/FormatDate';
+import { LOGIN_URL } from '@/constants/url';
+import useAuthenticated from '@/hooks/useAuthenticated';
+import StandUpContainer from '@/components/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+import { skipToken } from '@reduxjs/toolkit/dist/query';
+
+const StandUp: FC = () => {
+    const [standupUpdate, setStandupUpdate] = useState<standupUpdateType>({
+        type: 'user',
+        completed: '',
+        planned: '',
+        blockers: '',
+    });
+    const [buttonDisable, setButtonDisable] = useState<boolean>(true);
+
+    const { isLoggedIn, isLoading } = useAuthenticated();
+
+    const { isLoading: isAuthenticating } = useGetUserQuery(skipToken);
+
+    // here FormatDate() is a function that is defined inside the util folder and
+    // is used to calculate currentDate - 1 which returns the date as May 16, 2023
+    // which is getting stored in the yesterdayDate variable
+    const yesterdayDate = FormatDate();
+
+    const handleFormSubmission = (event: React.FormEvent<HTMLFormElement>) => {
```

Comment:
you are sending `handleFormSubmission` inside please check for rerender inside the StandupComponent since its being passed from here it will cause the child also to rerender everytime the state updates in this parent component

Reply:
I have tried to fix the issue please have a look, I have used React. Memo  to stop re-rendering of  FormInputComponent but still it re-renders the form I guess because  state is changing and when state changes re-render happens. Please let me know if there is anything else needs to be done.